File: 프로그래머스/3/42884. 단속카메라/단속카메라.js
function solution(routes) {
    routes.sort((a, b) => {
        return (a[1] === b[1]) ? (a[0] - b[0]) : (a[1] - b[1]);
    });
    
    let answer = 1;
    let last_camera = routes[0][1];
    
    routes.forEach(([a, b]) => {
        if (a > last_camera) {
            last_camera = b;
            answer++;
        }
    });
    
    return answer;
}

File: 프로그래머스/3/42884. 단속카메라/단속카메라.py
def solution(routes):
    routes = sorted(routes, key=lambda x: (x[1], x[0]))
    
    answer = 1
    last_camera = routes[0][1]
    
    for a, b in routes:
        if a > last_camera:
            last_camera = b
            answer += 1
    
    return answer
